frontend/src/components/WofInfoBox.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+    <wof-popper> <!--problem with WofPopper - doesnt work-->
+    <table :class="`WofBox WofBox--${this.type}`">
+        <tr :id="`first-row`">
+            <th :id="`title`">{{ title }}</th>
+            <th :id="`close-window`">
+                <wof-button :type="this.type" :id="`close-window-button`" @click="$emit('close')">
+                <wof-icon icon="wof-cancel" size=2  />
+                </wof-button>
+            </th>
+        </tr>
+        <tr>
+        <th colspan="2" id="content">{{ content }}</th>
```

Comment:
Use vue slot instead of this prop. We should be able to put everything we want into this box